samples/Mvc.Client/Startup.cs

```diff
@@ -45,6 +45,12 @@ public void ConfigureServices(IServiceCollection services)
                 options.Scope.Add("user:email");
             })
 
+            .AddGitLab(options =>
+            {
+                options.ClientId = "f4466c1688ecffbbcd9805cc0f0228c71da06d8d0554411259dfec3789711b59";
```

Comment:
Please remove this example and these credentials.

We don't want other peoples' API secrets in the code, and there's over 50 providers and they can't all be included in the sample app.

Reply:
Removed